github/ci/prow-deploy/files/jobs/kubevirt/containerized-data-importer/containerized-data-importer-presubmits.yaml

```diff
@@ -394,3 +394,241 @@ presubmits:
           resources:
             requests:
               memory: "4Gi"
+  - name: pull-containerized-data-importer-e2e-k8s-1.22-hpp
```

Comment:
So the way we have this configured right now there is no other way. But we could in theory do it another way. Similar to what I am doing in hpp. Instead of setting the env variables here. We make scripts in /hack or /automation that setup the variables. Then we configure these lanes to call those scripts. Then whenever we want to change the k8s version we can modify the scripts and leave the lanes alone. We would just have to find meaningful names for the lanes since they won't include the k8s version anymore.